acceptance/tests/environment/negative/agent_run_should_fail_if_env_unreadable.rb

```diff
@@ -0,0 +1,57 @@
+test_name "C97899 - Agent run should fail if environment is unreadable" do
+
+  testdir = ''
+  env_path = ''
+  test_env = ''
+
+  step 'setup environments' do
+    testdir = create_tmpdir_for_user master, 'c97899_unreadable_envdir'
+    env_path = "#{testdir}/environments"
+    test_env = "#{env_path}/testing"
+
+    apply_manifest_on(master, <<-MANIFEST, :catch_failures => true)
+      File {
+        ensure => directory,
+        mode => "0770",
+        owner => #{master.puppet['user']},
+        group => #{master.puppet['group']},
+      }
+      file {
+        '#{env_path}':;
+        '#{env_path}/production':;
+        '#{test_env}':;
+        '#{test_env}/manifests':;
+        '#{test_env}/modules':;
+      }
+      file { '#{test_env}/manifests/site.pp':
+        ensure => file,
+        mode => "0640",
+        content => 'node default { notify { "Hello agent": } }',
+      }
+    MANIFEST
+  end
+
+  step 'change permissions of envdir to 644' do
+    on(master, "chmod 644 #{test_env}")
+  end
+
+  step 'verify environment fails with puppet agent run' do
+    master_opts = {
+      'main' => {
+        'environmentpath' => env_path,
+      }
+    }
+    with_puppet_running_on master, master_opts, testdir do
```

Comment:
@joshcooper it looks like one of the pre-suite tasks for `ci:test:git` is currently broken on master. So, I am not able to successfully test that task.
```
Errored Tests Cases:
  Test Case setup/git/pre-suite/000_EnvSetup.rb reported: #<Beaker::Host::CommandFailure: Host 'ychcq6ueup73370.delivery.puppetlabs.net' exited with 1 running:
 cd /opt/puppet-git-repos/puppet-win32-ruby; cp -r ruby/* /
Last 10 lines of output were:
        bash: line 0: cd: /opt/puppet-git-repos/puppet-win32-ruby: No such file or directory
        cp: cannot stat 'ruby/*': No such file or directory>
    Test line: setup/git/pre-suite/000_EnvSetup.rb:119:in `block (2 levels) in run_test'
```
Filed [PUP-7376](https://tickets.puppetlabs.com/browse/PUP-7376)